bindings/go/evmc/evmc.go

```diff
@@ -391,6 +394,12 @@ func (vm *VMSteppable) StepN(params StepParameters) (res StepResult, err error)
 		stack = (*C.evmc_uint256be)(unsafe.Pointer(&params.Stack[0]))
 	}
 
+	pnr := new(runtime.Pinner)
+	pnr.Pin(&params.Context.GetTxContext().BlobHashes[0])
+	defer func() {
+		pnr.Unpin()
+	}()
```

Comment:
The pinning should be skipped if the BlobHashes are empty. Otherwise the access to element 0 fails.